src/api/addressTags.ts

```diff
@@ -0,0 +1,44 @@
+import { Client } from '../client';
+import { queue } from './utilities';
+
+export const addAddressTags = async (
+  tags: [{ [key: string]: string }],
+): Promise<Buffer> => {
+  // convert an array of objects to an object
+  const records = tags.reduce((acc, tag) => {
+    const key = Object.keys(tag)[0];
+    acc[key] = tag[key];
+    return acc;
+  }, {});
+
+  return queue((client) => client.addKvRecords({ records }));
+};
+
+export const fetchAddressTags = async () => {
+  const addresses: AddressTag[] = [];
+  let remainingToFetch = 10;
```

Comment:
Why the magic number? Shouldn't the starting index and number to fetch be function arguments?

Reply:
Swapped to use `MAX_ADDR` [235fd9e](https://github.com/GridPlus/gridplus-sdk/pull/525/commits/235fd9e7f44a6044fb7e5e6f850b64af23c5b4fc)